packages/web/app/src/pages/organization-settings.tsx

```diff
@@ -381,8 +381,9 @@ const SettingsPageRenderer = (props: {
                   <CardTitle>Organization Name</CardTitle>
                   <CardDescription>
                     Changing the name of your organization <span className="font-bold">won't</span>{' '}
```

Comment:
let's avoid contractions so it is more clear 
```suggestion
                    Changing the name of your organization will<span className="font-bold">not</span>{' '}
```